x/programs/rust/wasmlanche_sdk/src/store/mod.rs

```diff
@@ -139,20 +157,26 @@ fn get_field_as_bytes(ctx: &Context, name: &[u8]) -> Result<Vec<u8>, StorageErro
     let bytes_ptr = bytes_ptr as *mut u8;
 
     // Take ownership of those bytes grabbed from the host. We want Rust to manage the memory.
-    let bytes = unsafe { Vec::from_raw_parts(bytes_ptr, bytes_len as usize, bytes_len as usize) };
+    let bytes = unsafe {
+        Vec::from_raw_parts(
+            bytes_ptr,
+            bytes_len.try_into().unwrap(),
+            bytes_len.try_into().unwrap(),
```

Comment:
I know there's a check above assuring that the cast will succeed, but generally it's better to avoid using `as` casts with numbers unless you're doing something that's guaranteed to succeed (like `usize as u64`). 

I think there are some other improvements that can be made to this function (like calling `into_boxed_slice` so that you can't expand the `Vec`) and possible changing the return type for `get_bytes_len`, but that's for another PR.